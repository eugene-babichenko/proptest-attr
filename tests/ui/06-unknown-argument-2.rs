use proptest::prelude::*;
use proptest_attr::proptest;

fn main() {
    basic_test();
}

#[proptest(asdf)]
fn basic_test(_value: u8) -> Result<(), TestCaseError> {
    Ok(())
}
