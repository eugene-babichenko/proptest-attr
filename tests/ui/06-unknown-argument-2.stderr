error: unknown argument
 --> $DIR/06-unknown-argument-2.rs:8:12
  |
8 | #[proptest(asdf)]
  |            ^^^^

error[E0425]: cannot find function `basic_test` in this scope
 --> $DIR/06-unknown-argument-2.rs:5:5
  |
5 |     basic_test();
  |     ^^^^^^^^^^ not found in this scope

warning: unused import: `proptest::prelude::*`
 --> $DIR/06-unknown-argument-2.rs:1:5
  |
1 | use proptest::prelude::*;
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
